packages/govuk-frontend/src/govuk/all.mjs

```diff
@@ -1,124 +1,19 @@
-        console.log(error)
-      }
-    })
-  })
-}
-
-export {
-  initAll,
-  version,
-
-  // Components
-  Accordion,
-  Button,
-  CharacterCount,
-  Checkboxes,
-  ErrorSummary,
-  ExitThisPage,
-  Header,
-  NotificationBanner,
-  PasswordInput,
-  Radios,
-  SkipLink,
-  Tabs
-}
-
-/**
- * Config for all components via `initAll()`
- *
- * @typedef {object} Config
- * @property {AccordionConfig} [accordion] - Accordion config
- * @property {ButtonConfig} [button] - Button config
- * @property {CharacterCountConfig} [characterCount] - Character Count config
- * @property {ErrorSummaryConfig} [errorSummary] - Error Summary config
- * @property {ExitThisPageConfig} [exitThisPage] - Exit This Page config
- * @property {NotificationBannerConfig} [notificationBanner] - Notification Banner config
- * @property {PasswordInputConfig} [passwordInput] - Password input config
- */
-
-/**
- * Config for individual components
- *
- * @typedef {import('./components/accordion/accordion.mjs').AccordionConfig} AccordionConfig
- * @typedef {import('./components/accordion/accordion.mjs').AccordionTranslations} AccordionTranslations
- * @typedef {import('./components/button/button.mjs').ButtonConfig} ButtonConfig
- * @typedef {import('./components/character-count/character-count.mjs').CharacterCountConfig} CharacterCountConfig
- * @typedef {import('./components/character-count/character-count.mjs').CharacterCountTranslations} CharacterCountTranslations
- * @typedef {import('./components/error-summary/error-summary.mjs').ErrorSummaryConfig} ErrorSummaryConfig
- * @typedef {import('./components/exit-this-page/exit-this-page.mjs').ExitThisPageConfig} ExitThisPageConfig
- * @typedef {import('./components/exit-this-page/exit-this-page.mjs').ExitThisPageTranslations} ExitThisPageTranslations
- * @typedef {import('./components/notification-banner/notification-banner.mjs').NotificationBannerConfig} NotificationBannerConfig
- * @typedef {import('./components/password-input/password-input.mjs').PasswordInputConfig} PasswordInputConfig
- */
+export { version } from './common/govuk-frontend-version.mjs'
+export { Accordion } from './components/accordion/accordion.mjs'
+export { Button } from './components/button/button.mjs'
+export { CharacterCount } from './components/character-count/character-count.mjs'
+export { Checkboxes } from './components/checkboxes/checkboxes.mjs'
+export { ErrorSummary } from './components/error-summary/error-summary.mjs'
+export { ExitThisPage } from './components/exit-this-page/exit-this-page.mjs'
+export { Header } from './components/header/header.mjs'
+export { NotificationBanner } from './components/notification-banner/notification-banner.mjs'
+export { PasswordInput } from './components/password-input/password-input.mjs'
+export { Radios } from './components/radios/radios.mjs'
+export { SkipLink } from './components/skip-link/skip-link.mjs'
+export { Tabs } from './components/tabs/tabs.mjs'
+export { initAll, createAll } from './init.mjs'
```

Comment:
**note** With the removal of `initAll` from `all.mjs`, all the file does is re-exporting the difference pieces of our library that constitute our public API, so we can switch to the `export { NAMED_IMPORT } from 'module'` syntax rather than have a bunch of `import`s followed by an `export`.